### DISCUSSION TITLE
COPYLOT for students

### DISCUSSION THREAD

**Discussion by Shphilip** [community/community#113248] [GitHub Education]
### Hi everyone,

Hi, I applied for a free student COPYLOT, it says to me that the application was approved on one side and it says that it was rejected. In PYCHARM it, that there is no permission. What can I do?

**Comment by samus-aran** [comment]
Hi @Shphilip  👋

Ensure you have a successful GitHub Education [verification status](https://education.github.com/discount_requests/pack_application) and have received the confirmation email that your Academic benefits have been granted - this usually arrives 72 hours after you have been approved.

To enable Copilot, go to your account settings and under Code, planning, and automation section, select Copilot. You'll see "You're eligible to use copilot for free" and a button to claim the offer.

If you still unable to enable Copilot please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "I am a verified student having trouble redeeming my Global Campus offers". 👍

Hope this helps! Let us know how you get along 👍
